### PR TITLE
[F#] Extract Concepts from v2 exercise: meetup

### LINKED ISSUE #263: [F#] Extract Concepts from v2 exercise: meetup
Extract the list of concepts required to solve the v2 F# exercise: `meetup`. For each concept, describe why it is required to solve the exercise. Keep in mind that an exercise can have multiple approaches. The goal of this task is to help determine the total list of concepts that make up `meetup`.

The list of concepts should be stored in the following document: `languages/fsharp/reference/exercise-concepts/meetup.md`. [Use this link to create a PR to add this document][pr-link].

[pr-link]: ../new/master?filename=languages/fsharp/reference/exercise-concepts/meetup.md

### PR DESCRIPTION
**Pull request by ErikSchierboom**
Closes #263

### CODE REVIEW

File: languages/fsharp/reference/exercise-concepts/meetup.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Concepts of meetup
+
```
Comment (robkeim) [inline languages/fsharp/reference/exercise-concepts/meetup.md]: Can you add a link to the v2 example implementation for reference please?

File: languages/fsharp/reference/exercise-concepts/meetup.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+- properties: calling the `Day` property of a `DateTime`
+- return values: returning a value from a function
+- implicit returns: the last expression is automatically returned from a function
+- type inference: automatically infer the type of the functions and values
+- modules: the functions are defined in a module
+- imports: import types through `open` statements
+- namespaces: knowing where to find the `DateTime` class
+- pattern matching: matching on the `Week` discriminated union
+- assignment: assigning values
+- anonymous functions: a lambda is use to filter and map
```
Comment (robkeim) [inline languages/fsharp/reference/exercise-concepts/meetup.md]:
```suggestion
- anonymous functions: a lambda is used to filter and map
```

File: languages/fsharp/reference/exercise-concepts/meetup.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+- partial application: partially applying arguments to functions to return a new function
+- pipeline: using the `|>` operator to construct a pipeline
+- collection filtering: using `List.filter`, `List.find`, `List.last` and `List.item` to filter the list of dates
+- collection mapping: using `List.map` to map integers to `DateTime` instances
+- collection iteration: using `List` module functions to iterate over collection
```
Comment (robkeim) [inline languages/fsharp/reference/exercise-concepts/meetup.md]:
```suggestion
- collection iteration: using `List` module functions to iterate over collections
```

File: languages/fsharp/reference/exercise-concepts/meetup.md (hunk 1 above)
Comment (ErikSchierboom) [reply languages/fsharp/reference/exercise-concepts/meetup.md]: Done!

**Pull request merged**